### PR TITLE
added subtract function and test

### PR DESCRIPTION
**Pull request by apwillis1**

### CODE REVIEW

File: pythoncalculator/subtract.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def subtract(x, y):
+    return y - y
```
Comment (jasivan) [inline pythoncalculator/subtract.py]: I think there might be a typo here, and it should be 
```suggestion
    return x - y
```

Comment (bexzies) [inline pythoncalculator/subtract.py]: should be...
`def subtract(x, y):
    return x - y`

**Review by bexzies — CHANGES_REQUESTED** [review]
Please correct equation

**Review by jasivan — CHANGES_REQUESTED** [review]
Hi @apwillis1 , thanks for the great work, could you double check the subtract function please?

**Review by bexzies — APPROVED** [review]
AMAZING WORK

**Review by jasivan — APPROVED** [review]
@apwillis1, all good now, cheers!

**Pull request merged**